packages/components/organisms/f-checkout/CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.43.0
+-------------------------------
+*January 25, 2021*
+
+### Changed
+- The package now also exports its module, besides the component, so the consuming application can handle the registration of the Vuex module.
```

Comment:
Do you need to update anything in the readme to explain this change?

Reply:
Done now ✔️